Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,12 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array(), array(), array(), '', array(), array('POST'));
```

Comment:
Oh right I overlook the `master` part! Instead of cherry-picking, maybe just change the base?